common/php/class-module.php

```diff
@@ -215,15 +215,15 @@ function filter_posts_link( $slug, $post_type = 'post' ) {
 	 */
 	function enqueue_datepicker_resources() {
 
-		// Add the first day of the week as an available variable to wp_head
-		echo "<script type=\"text/javascript\">var ef_week_first_day=\"" . get_option( 'start_of_week' ) . "\";</script>";
```

Comment:
This line was making the PHPUnit tests around enqueuing noisy, so moved it to use `wp_add_inline_script`